packages/nextjs/app/builders/0x6C0FB309933931d111634921287aF902667032C4/page.tsx

```diff
@@ -0,0 +1,59 @@
+import Image from "next/image";
+import Link from "next/link";
+
+const DivyanshPage = () => {
```

Comment:
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.
```suggestion
const DivyanshPage: NextPage = () => {
```